documentation/book/assembly-kafka-mirror-maker-offset-commit-interval.adoc

```diff
@@ -0,0 +1,16 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-offset-commit-interval-{context}']
+
+= Auto commit interval
+
+You can specify the time interval for committing an offset after data are consumed from the source cluster.
+This interval is in milliseconds and by default is set to 60 000 ms.
```

Comment:
Should it be `60000` without space to make sure that doesn't break line in the middle?